src/actions/SendConfirmationActions.js

```diff
@@ -44,36 +44,7 @@ export type FioSenderInfo = {
   skipRecord?: boolean
 }
 
-export const newSpendInfo = (spendInfo: EdgeSpendInfo, authRequired: AuthType) => ({
-  type: 'UI/SEND_CONFIMATION/NEW_SPEND_INFO',
-  data: { spendInfo, authRequired }
-})
-
-export const reset = () => ({
-  type: 'UI/SEND_CONFIMATION/RESET'
-})
-
-export const updateTransaction = (transaction: ?EdgeTransaction, guiMakeSpendInfo: ?GuiMakeSpendInfo, forceUpdateGui: ?boolean, error: ?Error) => ({
-  type: 'UI/SEND_CONFIMATION/UPDATE_TRANSACTION',
-  data: { transaction, guiMakeSpendInfo, forceUpdateGui, error }
-})
-
-export const updateSpendPending = (pending: boolean) => ({
-  type: 'UI/SEND_CONFIMATION/UPDATE_SPEND_PENDING',
-  data: { pending }
-})
-
-export const newPin = (pin: string) => ({
-  type: 'UI/SEND_CONFIMATION/NEW_PIN',
-  data: { pin }
-})
-
-export const toggleCryptoOnTop = () => ({
-  type: 'UI/SEND_CONFIMATION/TOGGLE_CRYPTO_ON_TOP',
-  data: null
-})
-
-const updateAmount =
+export const updateAmount =
```

Comment:
could you check, do we need to export this? not exporting in previous version

Reply:
Good catch. Merge conflict messed this up.